### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by sh1zicus** [sh1zicus/dots-hyprland-old#36] [Q&A]
Ask the community for help

**Comment by PharmaRacist** [comment]
how is the work for the messaging sidebar

**Comment by sh1zicus** [comment]
lol after changing the music the color scheme of pywal also changes, needs to be fixed 😆

**Reply by sh1zicus** [reply]
> lol after changing the music the color scheme of pywal also changes, needs to be fixed 😆

only vscode change colors

**Reply by PharmaRacist** [reply]
i dont understaaaand

**Reply by sh1zicus** [reply]
> i dont understaaaand


https://github.com/user-attachments/assets/94ef4fd8-4cf1-4a97-a185-f6ce4594f445

**Reply by PharmaRacist** [reply]
i think its problem with the original color gen stuff for music control maybe it interfere with our pywal

**Reply by sh1zicus** [reply]
> i think its problem with the original color gen stuff for music control maybe it interfere with our pywal

I solved the problem, in the last commit I removed the color generation in the music module and the cover, and now after switching music ags does not slow down

**Reply by PharmaRacist** [reply]
thats good to hear the music cover stuff is laggy any ways but i need to make it more vibrant to pop

**Comment by PharmaRacist** [comment]
im working on adding multiple bar modes and i want to make the bar corners attached to each bar mode like enabled for normal disabled for focused  can you help me do it ?

**Comment by PharmaRacist** [comment]
https://github.com/user-attachments/assets/48c87fdb-6a86-4487-b15d-75c3dca02472

**Comment by PharmaRacist** [comment]
i got it working  😀
Uploading 2024-12-09 20-53-48.mp4…

**Comment by PharmaRacist** [comment]
actually no i need help 😭

**Reply by sh1zicus** [reply]
I'm not in a position to help right now

**Comment by PharmaRacist** [comment]
so what do you think of ags v2 end4 seems to not caring about it

**Comment by PharmaRacist** [comment]
https://streamable.com/7vo61k 
im really thinking to bring this back with ags

**Comment by nichroot** [comment]
honestly would be kind of cool if end's old eww windows rice was ported to
ags too as sort of an undercover mode for this

**Reply by PharmaRacist** [reply]
my approach for multiple bars made me able to do similer one to novel knock but ags is currently lacking some animations or its just i dont know 🐸

**Comment by Tacocat263** [comment]


**Reply by PharmaRacist** [reply]
just sudo apt install

**Comment by Viator-Ryubei** [comment]
I'd like to know how to get notifications about certain repository so I can keep tabs on bug fixes and patches. How can I do that?

**Reply by PharmaRacist** [reply]
sh1zicus did a sideleft mpdule that constantly update the commits u can see that and ill make a bar module for that when i have time for that

**Reply by Viator-Ryubei** [reply]
Thank you for letting me know.
